src/js/components/locations-configuration/LocationDetails.jsx

```diff
@@ -362,7 +369,7 @@ class LocationDetails extends Component {
         fgColor: values.fgColor || '',
       };
 
-      apiClient.post(locationUrl, payload)
+      apiClient.post(locationUrl, payload, { params: { step: 1 } })
```

Comment:
Not a huge fan of this params thing. Should probably be part of the payload, right?

@awalkowiak Any thoughts?